_posts/2015-07-14-learning-from-our-failures.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+name: learning-from-our-failures
+title: Learning from our failures
+date: 2015-07-14 17:00:00 +00:00
+author: Franziska Sauerwein
+image:
+    src: /assets/img/custom/blog/failure.jpg
+    attribution:
+        text: Ahhhhhhh CC-BY Kenny Louie
+        href: https://flic.kr/p/88qwuz
+tags:
+- failure
+- learning
+- github
+- travis
+- company culture
+---
+
+On my second day at Codurance, I managed to delete the repository with [Codurance's Github Pages](http://codurance.com/) on it. This made the website unavailable for the first time in a long time, and during best business hours.
```

Comment:
Might be worth linking that to the actual repo rather than Codurance's home page.